#!/usr/bin/env python
PACKAGE="vn_100"
from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()
gen.add("kp_depth",double_t,0,"proportional depth parameter",0,0,300)
gen.add("ki_depth",double_t,0,"integral depth parameter",0,1,300)
gen.add("kd_depth",double_t,0,"derivative depth parameter",0,1,300)
gen.add("setpoint",double_t,0,"depth_setpoint",1000,1000,1300)
gen.add("bool_param",bool_t,0,"to time the values",False)
exit(gen.generate(PACKAGE,"control","depthparams"))